### PR TITLE
Fixed deprecated code after the release of WTForms 3.0

### LINKED ISSUE #2961: Deprecated wtforms.TextField and problem with flask-ldap3-login after WTForms upgrade
TextField was removed in wtforms v.3.0.0 (released yesterday Nov 7th). Now all the imports of this element are broken and should be replaced by wtforms.StringField. https://wtforms.readthedocs.io/en/3.0.x/whats_new/?highlight=textfield

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #2961.

**How to test**:
1. Install on clinical-dd stage and test some cases

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by DN

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -8,6 +8,7 @@ Flask-Bootstrap
 Flask-CORS
 path.py
 Flask-Markdown
+WTForms<3
```
Comment (dnil) [inline requirements.txt:11]: 🥶 👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,14 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
+## []
+- Use of deprecated TextField after the upgrade of WTF to v3.0
+- Freeze to WTForms to version < 3
+
+## [4.41.1]
+### Fixed
+- General reports crash for variant annotations with same variant on other cases
```
Comment (dnil) [inline CHANGELOG.md:13]: Hm, seems you have merged the hot fix into this already?!

File: scout/adapter/mongo/variant.py [hunk 3]
```diff
@@ -780,6 +780,7 @@ def evaluated_variants(self, case_id, institute_id):
             "$and": [
                 {"variant_id": {"$in": variant_ids}},
                 {"institute": institute_id},
+                {"case_id": case_id},
```
Comment (dnil) [inline scout/adapter/mongo/variant.py:783]: Ah yes.

**Review by dnil — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (northwestwitch) [reply CHANGELOG.md:13]: Yeah, it was the branch I've started this PR from, because I didn't realized the fix was released yet!

**Comment by dnil** [comment]
Ok, the extra files are actually blocking a smooth update.
`error: The following untracked working tree files would be overwritten by checkout:
	scout/server/blueprints/cases/static/madeline.svg
Please move or remove them before you can switch branches.`

**Comment by northwestwitch** [comment]
Merging this as soon as it passes the tests!

**Comment by northwestwitch** [comment]
My PR to fix lask-ldap3-login was merged yesterday: https://github.com/nickw444/flask-ldap3-login/pull/96#event-5593821070. The problem is that it will be available from next release (a major), that is likely going to have backward compatibility issues. Let's see what changes this version introduces, perhaps it's useful stuff for us as well!

**Pull request merged**